src/extension.ts

```diff
@@ -188,7 +186,8 @@ class RunLineInEditor {
     }
 
     private findResultDocument() {
-        if (this.resultDocument) {
+        var showResponseInDifferentTab = workspace.getConfiguration("ms-azurecli").get<boolean>("showResponseInDifferentTab", false)
```

Comment:
We use 'const' instead of the older 'var' (or 'let' if it can change). Use single quotes ' instead of double-quotes ".

Reply:
Fixed - thanks!